qutebrowser/browser/urlmarks.py

```diff
@@ -186,7 +186,7 @@ def set_mark():
 
         if name in self.marks:
             message.confirm_async(
-                title="Override existing quickmark?",
+                title="Such quickmark already exists for this URL {}.\nOverride existing quickmark?".format(self.marks[name]),
```

Comment:
I feel like the new error message implies that another quickmark already exists for the same URL - but that wouldn't be a problem. What about something like this?

```suggestion
                title=f"Override existing quickmark?\nOld URL: {self.marks[name]}"
```

(also using [f-strings](https://fstring.help/) for more modern string formatting - we don't use it consistently yet, but it should be preferred for new code)

Reply:
I initially did it with f-string, but then saw the formatting in the file and decided to make it the same for consistency.
If you think that useful I can start refactoring the files to f-string in my free time.

Reply:
Did I cancel the your suggestion by mistake when I corrected the typo in my commit? I did this  https://github.com/qutebrowser/qutebrowser/pull/7635#issuecomment-1479147811 in order to correct the typo in the commit message.